docs/plugins.asciidoc

```diff
@@ -43,10 +37,11 @@ You can specify URLs that use the HTTP, HTTPS, or `file` protocols.
 [float]
 === Installing Plugins to an Arbitrary Directory
 
-Use the `-d` or `--plugin-dir` option to specify a directory for plugins, as in the following example:
+Use the `-d` or `--plugin-dir` option after the `install` command to specify a directory for plugins, as in the following 
+example:
 
 [source,shell]
-bin/kibana plugin -i elasticsearch/sample-plugin/latest -d <path/to/directory>
+$ bin/kibana-plugin install elasticsearch/sample-plugin/latest -d <path/to/directory>
```

Comment:
Should just be plugin name, and no `/latest`.  Double check that there are no references to org name or things like `/latest` throughout this doc.  Plugin name or url only.